common/lib/xmodule/xmodule/course_module.py

```diff
@@ -357,7 +357,7 @@ class CourseFields(object):
     html_textbooks = List(
         display_name=_("HTML Textbooks"),
         help=_(
-            "For HTML textbooks that appear as separate tabs in the courseware, enter the name of the tab (usually "
+            "For HTML textbooks that appear as separate tabs in the course, enter the name of the tab (usually "
             "the name of the book) as well as the URLs and titles of all the chapters in the book."
```

Comment:
only if you have time: 
change this second use of "name" to "title"
all the chapters --> each chapter